pcsx2/gui/MainFrame.cpp

```diff
@@ -803,20 +817,17 @@ void MainEmuFrame::CommitPreset_noTrigger()
 	g_Conf->EmuOptions.EnablePatches = menubar.IsChecked(MenuId_EnablePatches);
 }
 
-static void AppendShortcutToMenuOption(wxMenuItem& item, wxString keyCodeStr)
+void MainEmuFrame::AppendShortcutToMenuOption(wxMenuItem& item, wxString keyCodeStr)
 {
+	if (&item == nullptr)
+	{
+		return;
+	}
```

Comment:
This return statement will never get hit due to how pass-by-reference variables of non-pointer types work.

Reply:
This PR went through various revisions, i think it's fine if i delete those 4 lines if i'm not mistaken. Have other stuff to do till end of the year, if it's a small fix i can do it still.